fmf/base.py

```diff
@@ -103,11 +106,11 @@ def __init__(self, data, name=None, parent=None):
         else:
             self.root = self.parent.root
             self.name = os.path.join(self.parent.name, name)
-        # Initialize data
-        if isinstance(data, dict):
-            self.update(data)
-        else:
+        # Update data from directory path (string) or data dictionary
+        if isinstance(data, type("")):
```

Comment:
Why you use `type("")` and not `str` directly, is there any  rule, why this is better?

Reply:
That's because of python 2/3 compatibility.

Reply:
Ah, thanks for explanation. I though it is some reason like this.